packages/common-types/src/index.ts

```diff
@@ -91,6 +89,25 @@ export interface TransactionActionApproveTokenData {
   isPermit2Enabled?: boolean;
 }
 
+export enum SignStatus {
+  none = 'none',
+  signed = 'signed',
+  failed = 'failed',
+}
+
+export interface TransactionActionApproveTokenSignDCAData {
+  signStatus: SignStatus;
+  simulation?: BlowfishResponse;
+  from: Token;
+  sellAmount: bigint;
```

Comment:
this is actually only relevant for swap

Reply:
I'm seeing amount as a required field for `getPermit2DcaSignedData`. Im renaming this variables to work for DCA and Swap concepts.
Also I have included a SuccessLabel for DCA Sign Step saying 'Invest X TKN' as we do for approve and for sign in Swap

![Screenshot 2024-02-14 at 13 30 15](https://github.com/Mean-Finance/dca-fe/assets/144699868/a1d7bf4b-7cc5-487e-9d62-643219c24077)

Reply:
Mmm i think i would put "Token authorization signed" instead of the amount